packages/jellyfish-transaction-builder/src/txn/txn_builder_token.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  OP_CODES, Script, TransactionSegWit, TokenBurn
+} from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+
+export class TxnBuilderTokens extends P2WPKHTxnBuilder {
+  /**
+   * Burn tokens
+   *
+   * @param {TokenBurn} tokenBurn txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+  async burnTokens (tokenBurn: TokenBurn, changeScript: Script): Promise<TransactionSegWit> {
```

Comment:
```suggestion
  async burn (tokenBurn: TokenBurn, changeScript: Script): Promise<TransactionSegWit> {
```

How about just calling it `burn`, since this class already has the world `tokens` in `TxnBuilderTokens`?